### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by Kamasah-Dickson** [community/community#63832] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Projects

### Body

I tried to connect my github project to vercel and after authentication I got a message by github saying

You are marked as spam, and therefore cannot authorize a third party application.
please kindly help me out

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by MINEGHOST007** [comment]
found any solution ??
